AdventOfCode.Library/Day10/Part2.fs

```diff
@@ -0,0 +1,32 @@
+module AdventOfCode.Library.Day10.Part2
+
+open AdventOfCode.Library.Day10.Part1
+
+let encode (input: bool seq) : int =
+    input
+    |> Seq.map (fun b -> if b then 1 else 0)
+    |> Seq.mapi (fun i n -> n * pown 2 i)
+    |> Seq.sum
+
+let identify (input: bool seq seq) : char option =
+    match input |> Seq.map encode |> Seq.toArray with
+    | [| 49; 41; 37; 35; 0 |] -> Some 'Z'
```

Comment:
This works for the letters in my solution. Expanding it to support other letters would require knowing what all the letters look like.